File: Java/Ten.java
package JAVA;
//import java.util.*;

public class Ten {
    public static boolean isValid(String s) {
        /*
         Enter code here
         */
        return false;
    }

    // Driver code
    public static void main(String[] args) {
        System.out.println(isValid("()"));        // Output: true
        System.out.println(isValid("()[]{}"));    // Output: true
        System.out.println(isValid("(]"));        // Output: false
        System.out.println(isValid("([)]"));      // Output: false
        System.out.println(isValid("{[]}"));      // Output: true
    }
}

File: Python/Ten.py
def isValid(s: str) -> bool:
    '''
    enter code here
    '''

# Driver code
if __name__ == "__main__":
    print(isValid("()"))        # Output: True
    print(isValid("()[]{}"))    # Output: True
    print(isValid("(]"))        # Output: False
    print(isValid("([)]"))      # Output: False
    print(isValid("{[]}"))      # Output: True
